### DISCUSSION TITLE
Strange spacing CSS bug in blog excerpts

### DISCUSSION THREAD

**Discussion by soranauts** [arthelokyo/astrowind#332] [Q&A] [answered]
Would anyone know of a way to fix this and does it maybe have something to do with Tailwind?

![Screenshot 2023-12-25 at 6 16 52 PM](https://github.com/onwidget/astrowind/assets/143734186/1335e6dc-0684-4157-bb00-c7ca8f3f0245)

**Comment by rhuddlestone** [comment, marked as answer]
If you look in the SinglePost.astro component you will see

p
        class="max-w-3xl mx-auto mt-4 mb-8 px-4 sm:px-6 text-xl md:text-2xl text-muted dark:text-slate-400 text-justify"
      >
        {post.excerpt}
      </p>

The text-justify option is causing this spacing.  Try removing the text-justify and it should all justify to the left with regular spacing.

**Comment by soranauts** [comment]
> If you look in the SinglePost.astro component you will see
> 
> p class="max-w-3xl mx-auto mt-4 mb-8 px-4 sm:px-6 text-xl md:text-2xl text-muted dark:text-slate-400 text-justify" > {post.excerpt}
> 
> The text-justify option is causing this spacing. Try removing the text-justify and it should all justify to the left with regular spacing.

You're totally right. That worked perfectly. Thanks so much!

_Leaving this open so they can update the theme if needed._
